### PR TITLE
Amass Tokens not using the appropriate names for images

### LINKED ISSUE #4344: Question about the naming convention for the Army tokens
**Describe the bug**
Noticed recently that after the amass change with the LTR set the army tokens no longer show up with an image, nor does Forge download a new image. Just wondering what the new naming rule for the tokens is, so I can manually update them.

### PR DESCRIPTION
**Pull request by tehdiplomat**
Closes #4344

### CODE REVIEW

**Comment by Hanmac** [comment]
No that doesn't work!

Card Text Changes would BREAK it

**Comment by Hanmac** [comment]
like i commented there: #4344

> @kevlahnota we need to update the army token list in the token images

**Review by Hanmac — CHANGES_REQUESTED** [review]
Card Text Changes would break it

**Comment by tehdiplomat** [comment]
@Hanmac Tested with Change type and its working exactly how i expect.

**Comment by Hanmac** [comment]
I think the Original Param might still be altered if a Token of the Card is created 🤔 

I need to check that out

**Review by Hanmac — DISMISSED** [review]
i think this is okay for now until i find a way to break it again

File: forge-gui/res/tokenscripts/b_0_0_zombie_army.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Zombie Army Token
```
Comment (tool4ever) [inline forge-gui/res/tokenscripts/b_0_0_zombie_army.txt:1]: so we need one for each type now?
what about _Lazotep Sliver_?

Comment (tehdiplomat) [reply forge-gui/res/tokenscripts/b_0_0_zombie_army.txt:1]: Yea, but these scripts should already have been unique since they have different names.

**Review by tool4ever — APPROVED** [review]

**Review by Hanmac — CHANGES_REQUESTED** [review]
Wouldn't it be better to keep the TokenScript and just alter the Image Key?

**Comment by tehdiplomat** [comment]
@Hanmac Why are you blocking this now again? The token is not named just army.

**Comment by Hanmac** [comment]
> @Hanmac Why are you blocking this now again? The token is not named just army.

I just say it would have been easier to just manipulate the image key instead of creating extra token scripts

**Pull request merged**